### PR TITLE
LPS-134298 There is no space between two fields group when they are side by side on the published form

### PR DESCRIPTION
**Pull request by mateusgomes01**

### CODE REVIEW

**Comment by mateusgomes01** [comment]
@miltonmc Milton, could you please review this pull request? Thanks!

File: modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/css/main.scss [hunk 1]
```diff
@@ -43,6 +43,10 @@
 }
 
 body .ddm-user-view-content {
+	.ddm-field-container {
```
Comment (miltonmc) [inline modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/css/main.scss:46]: Try to keep classes as flat as possible.

Some useful information:
- https://thesassway.com/avoid-nested-selectors-for-more-modular-css/

File: modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/css/main.scss [hunk 2]
```diff
@@ -43,6 +43,10 @@
...
+		padding: 0 12px;
```
Comment (miltonmc) [inline modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/css/main.scss:47]: I think you should use 8 base values (like 8, 16, 24, ...) on portal's style

**Review by miltonmc — CHANGES_REQUESTED** [review]

File: modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/css/main.scss (hunk 2 above)
Comment (mateusgomes01) [reply modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/css/main.scss:47]: Hey @miltonmc , thank you for your feedback!

In this case I used 12px to keep consistency with the padding present in the form builder:

https://github.com/liferay/liferay-portal/blob/3a8ab55f0466afd7405223636a5cef15307a31dd/modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/css/form-builder/_form-builder.scss#L31

Also, 12px was the previous value used on the field padding in previous versions of the portal. Do you think we should we change both to 8px/16px or keep as 12px? Or something else?

File: modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/css/main.scss (hunk 1 above)
Comment (mateusgomes01) [reply modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/css/main.scss]: Thank you for the insight! I have updated the code to reflect these better css practices

**Comment by mateusgomes01** [comment]
ci:test:sf
ci:test:relevant

**Comment by mateusgomes01** [comment]
ci:test:sf
ci:test:relevant

**Review by miltonmc — APPROVED** [review]

**Comment by miltonmc** [comment]
ci:forward

**Pull request closed without merging**